server/errorHandler.ts

```diff
@@ -11,14 +11,21 @@ export default function createErrorHandler(production: boolean) {
       return res.redirect('/sign-out')
     }
 
+    const heading = error.status === 404 ? 'Page not found' : 'Error'
     const prodMessage =
```

Comment:
Happy to approve, but I guess if being picky on naming this would probably now be better named just `message` to avoid confusion